### ISSUE TITLE
Add Desktop build preview in PRs using a pipeline

### ISSUE THREAD

**Issue by juans-chainsafe** [ChainSafe/web3.unity#1244]
**Summary**
We want to generate a preview of a Desktop build in the PRs so we know if the build is working and we save time for QA tasks

**Suggested features or ways to address the summary**
- Generate the build and display it on the PR so people can access and use it
- Not sure how this will work for Mac and Desktop, are the same build? or they are different depending on the platform? @kantagara @rob1997 @oleksandrchainsafe @sneakzttv 
- Take in mind, that we need to add the parameters needed for Reown and Web3Auth like project Id in order for them to work

**Motivation**
This can also be something that we can use to add e2e tests in the build that was generated

**Acceptance Criteria**
- Desktop build is generated
- Build is displayed in the PR
- Build can be downloaded and install it on the laptop
- Login and samples works as they will work locally

**Comment by kantagara** [comment]
@juans-chainsafe yeah we would need to generate two different builds on two different machines.
